joomla/joomla-cms
# [5.0] core extensions remove captcha plugins

Pull Request for Issue https://github.com/joomla/joomla-cms/pull/41530#issuecomment-1699616466 .

### Summary of Changes

> It should the also be removed from the list of core extensions in the ExtensionsHelper

### Testing Instructions

update from 4.4 to 5.0 nightly and check core extensions
![image](https://github.com/joomla/joomla-cms/assets/66922325/c2dd5e1f-26b6-44d8-882e-40d5239eabe1)

### Actual result BEFORE applying this Pull Request

captcha plugin(s) are still listed as core extension

### Expected result AFTER applying this Pull Request

captcha plugins are no longer listed as core extension

### Link to documentations
Please select:
